gateway-shell/src/main/java/org/apache/knox/gateway/shell/table/KnoxShellTableFilter.java

```diff
@@ -82,4 +84,23 @@ public KnoxShellTable greaterThan(Comparable<? extends Object> comparable) throw
     return filter(s -> s.compareTo(comparable) > 0);
```

Comment:
I think you can store the predicate as a variable name and reuse it.

Something like:

`Predicate greaterThan = s -> s.compareTo(comparable) > 0;`

Might have to fix it up a bit but it would make the new methods simpler since you could say `greaterThan(s, comparable) || equalTo(s, comparable)`

Reply:
I like the idea but I can't find away to implement this in a fashion that would be worth the effort.